programmers2/게임-맵-최단거리.js

```diff
@@ -0,0 +1,28 @@
+function solution(maps) {
+    const yLen = maps.length;
+    const xLen = maps[0].length;
+    const dx = [-1, 1, 0, 0];
+    const dy = [0, 0, -1, 1];
+    const queue = [[0, 0]];
+    
+    while (queue.length) {
+        let xy = queue.shift();
+        let x = xy[0];
+        let y = xy[1];
```

Comment:
```suggestion
        const xy = queue.shift();
        const x = xy[0];
        const y = xy[1];
```
const 로 선언해도 정상 작동합니다 잘은 모르겠지만 아마 
한 번 반복할 때마다 새로운 변수로 선언하는 것처럼 작용하는게 아닐까 싶어요